src/modules/auth/resources/ResourcesDeletedDetailContainer.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+
+import {
+	Container, Row, Col, Card,
+	CardHeader, CardBody, Button,
+	CardFooter, ButtonGroup
+} from 'reactstrap';
+
+import { DateTime, ButtonWithAuthz } from 'asab-webui';
+
+const ResourceDetailContainer = (props) =>  {
+	const { t } = useTranslation();
+	const SeaCatAuthAPI = props.app.axiosCreate('seacat_auth');
+	const [resource, setResource] = useState(null);
+	const [editMode, setEditMode] = useState(false);
+	const { resource_id } = props.match.params;
+
+	const resources = useSelector(state => state.auth?.resources);
+
+	const getResourceDetail = async (res) => {
+		try {
+			let response = await SeaCatAuthAPI.get(`resource/${res}`);
+			setResource(response.data);
+		} catch(e) {
+			console.error(e);
+			props.app.addAlert("warning", `${t("ResourcesDeletedDetailContainer|Can't fetch resource details")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
+
+	useEffect(() => {
+		getResourceDetail(resource_id);
+	}, []);
+
+	if (!resource) return null;
```

Comment:
@petrKavulok Shouldnt this be wrapped at least in some useEffect? This is very unefficient. Or why it is here in the first place?